articles/language-negotiation/language-negotiation.md

```diff
@@ -203,26 +218,29 @@ outside that region.
 
 Notice that there is a tension between providing a long list of supported locales
 (to give users the ability to tailor API-based formatting presentation)
-and providing a short list of available languages.
+and providing a short list of available locales (to aid in selection).
 In our example above, where the site is available in English, French, and Spanish,
-there might only be three localizations, but twenty or more locales.
+there might only be three localizations, but twenty or more locales that the site _could_
+make available.
+Deciding which combinations of language and locale to expose to users and how to represent these
+depends on many factors.
 
 ## Constructing the Algorithm
 
-Hierarchical negotiation is the most common mechanism for performing language negotiation.
+Hierarchical negotiation is the most common mechanism for performing locale negotiation.
 One way to implement this is to work from the least specific signal to the
 most specific one and then return the result.
 
 > For example:
 >
 > 1. Let the return value be the site default.
-> 2. If the user's geographic region has a default let return value be that language.
+> 2. If the user's geographic region has a default let return value be that locale.
 > 3. If the user has an Accept-Language header
->    i. For each language in the A-L header
->       a. if the language matches an available language, let return value be that language
-> 4. If the user has a cookie with a language preference, let return value be that language
-> 5. If the URL contains a language, let return value be that language
-> 6. If the user is recognized, let return value be the language in the user's profile
+>    i. For each language range in the A-L header
+>       a. if the language range matches an available locale's language tag, let return value be that locale
```

Comment:
nit: Markdown doesn't seem to support list syntax like `i.` or `a.`.

Reply:
Yes: the article will be converted to HTML before final publication.